### DISCUSSION TITLE
Alternate and Single Tap mods for Taiko?

### DISCUSSION THREAD

**Discussion by OpenSauce04** [ppy/osu#22630] [Ideas] [answered]
I feel that these mod could be implemented into Taiko quite seamlessly
For Single Tap, make it so that the first input for both Don and Kat is the only one you can use for that specific action
For Alternate, make it so that you have to alternate between keys on Don and Kat independently

The only issue I can think of is how to deal with strong hits
You could possibly just allow the player to use both the left and right action only when hitting a strong hit?

**Comment by mk56-spn** [comment]
https://github.com/ppy/osu/pull/7953#issuecomment-591357112

**Reply by OpenSauce04** [reply]
So from what I can surmise, Taiko was originally planned for the mod, but the implementation was too bad?
I don't see any point where someone said it couldn't be a thing, just that the approach needed to be different

**Reply by mk56-spn** [reply]
neither did i say that, im simply saying theres already been an attempt, and theres also already a discussion open 

https://github.com/ppy/osu/discussions/18086

**Reply by OpenSauce04** [reply]
Ah, I did look for an existing discussion, but couldn't find it. Guess my search term was off. Thanks!

**Comment by OpenSauce04** [comment, marked as answer]
(Sort of) Duplicate of #18086
